submissions/alstep07/html-css-popup/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
```

Comment:
The single tags should be without slashes "/", could you please fix it for all single tags?

Reply:
fixed